fix: NetApp - Added missing param passthru

AlexanderSehr opened the pull request:
## Description

Added missing param passthru

cc: @Salima-90 (thanks for raising the issue)

## Pipeline Reference

<!-- Insert your Pipeline Status Badge below -->

| Pipeline |
| -------- |
|    [![avm.res.net-app.net-app-account](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.net-app.net-app-account.yml/badge.svg?branch=users%2Falsehr%2FnetAppMissingVolumeType&event=workflow_dispatch)](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.net-app.net-app-account.yml)      |

## Type of Change

<!-- Use the checkboxes [x] on the options that are relevant. -->

- [ ] Update to CI Environment or utilities (Non-module affecting changes)
- [x] Azure Verified Module updates:
  - [x] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

Salima-90 commented:
Hi @AlexanderSehr, thank you for raising the PR, it's much appreciated. 

I was due to raise a PR to update the api to the latest 2024-07 versions as we are testing with volume Type set to 'Migration' which only works with the latest API. I was wondering to avoid conflicts if this can be added to your PR?

Also, I just wish to clarify if kerberos is enabled as we notice you have added a change to enable Kerberos but even with testing locally this does not work as intended? 

Please can you also provide an expected timescale of when this will be merged as we are under pressure to get data migrated to ANF.

Thank you

AlexanderSehr commented:
> Hi @AlexanderSehr, thank you for raising the PR, it's much appreciated.
> 
> I was due to raise a PR to update the api to the latest 2024-07 versions as we are testing with volume Type set to 'Migration' which only works with the latest API. I was wondering to avoid conflicts if this can be added to your PR?
> 
> Also, I just wish to clarify if kerberos is enabled as we notice you have added a change to enable Kerberos but even with testing locally this does not work as intended?
> 
> Please can you also provide an expected timescale of when this will be merged as we are under pressure to get data migrated to ANF.
> 
> Thank you

Hey @Salima-90,
I just updated the API version. Regarding `kerberosEnabled`, you can configure it as `true` when defining the `volume`. By default it is deployed with a value of `false`.
Finally, regarding the timeline, it depends on the module owner @fbinotto. To accelerate, I'll reach out internally too :)

bobmclane999 commented:
Hey @fbinotto / @jtracey93 been a while :) , hope you guys are well...!? 
If you wouldn't mind giving Alexander's change the once over and approving as soon as poss that would be awesome for us as we are working closely with the Anthony (NetApp) to get the new volumes with "Migration" set so we can start migrating our data over to ANF in Azure for our DC Exit project..

Thanks as always fellas...

Cheers
Shaun

Salima-90 commented:
> > Hi @AlexanderSehr, thank you for raising the PR, it's much appreciated.
> > I was due to raise a PR to update the api to the latest 2024-07 versions as we are testing with volume Type set to 'Migration' which only works with the latest API. I was wondering to avoid conflicts if this can be added to your PR?
> > Also, I just wish to clarify if kerberos is enabled as we notice you have added a change to enable Kerberos but even with testing locally this does not work as intended?
> > Please can you also provide an expected timescale of when this will be merged as we are under pressure to get data migrated to ANF.
> > Thank you
> 
> Hey @Salima-90, I just updated the API version. Regarding `kerberosEnabled`, you can configure it as `true` when defining the `volume`. By default it is deployed with a value of `false`. Finally, regarding the timeline, it depends on the module owner @fbinotto. To accelerate, I'll reach out internally too :)

Apologies, @AlexanderSehr  I forgot to mention but for migration volume types additional parameters would need to be added in the latest volume type module 
<img width="433" alt="image" src="https://github.com/user-attachments/assets/b8184679-2fc4-4365-8161-943a34d39df4" />

AlexanderSehr commented:
> > > Hi @AlexanderSehr, thank you for raising the PR, it's much appreciated.
> > > I was due to raise a PR to update the api to the latest 2024-07 versions as we are testing with volume Type set to 'Migration' which only works with the latest API. I was wondering to avoid conflicts if this can be added to your PR?
> > > Also, I just wish to clarify if kerberos is enabled as we notice you have added a change to enable Kerberos but even with testing locally this does not work as intended?
> > > Please can you also provide an expected timescale of when this will be merged as we are under pressure to get data migrated to ANF.
> > > Thank you
> > 
> > 
> > Hey @Salima-90, I just updated the API version. Regarding `kerberosEnabled`, you can configure it as `true` when defining the `volume`. By default it is deployed with a value of `false`. Finally, regarding the timeline, it depends on the module owner @fbinotto. To accelerate, I'll reach out internally too :)
> 
> Apologies, @AlexanderSehr I forgot to mention but for migration volume types additional parameters would need to be added in the latest volume type module. Also, I notice the volume module has not been updated to the latest api 

Hey @Salima-90,
I just added the properties and re-triggered the pipeline.

fbinotto reviewed (approved):
LGTM!
